src/density-qvm.lisp

```diff
@@ -124,7 +124,7 @@ recorded outcome j may be different."))
       (let ((density-qvm (apply #'make-instance 'density-qvm
                                 :number-of-qubits num-qubits
                                 :amplitudes amps
-                                initargs)))
+                                (alexandria:remove-from-plist initargs ':allocation))))
```

Comment:
NB: this change is in the QVM package, not QVM-APP-NG.

`:allocation` is not a valid initarg for `density-qvm`, so strip it prior to calling `make-instance`.

Reply:
This change is no longer needed due to the recent state rep refactoring.